uwblueprint/jump-math
# fix: remove class field from teacher (2)

## Notion ticket link

[Remove class field from teacher](https://www.notion.so/uwblueprintexecs/Remove-class-field-from-teacher-c63d70810ec84ce18df96cc569170049?pvs=4)



## Implementation description
* Remove `class` field from User model since it is unused
* Update createClass logic to not add class to teacher
* Fix unit tests to support new model


## Steps to test
1. Verify that creating a class works
2. Verify that classrooms page, classrooms section in dashboard, and distribute assessment flow works



## What should reviewers focus on?
* No breaking changes


## Checklist
